resolver/resolver.cc

```diff
@@ -2553,8 +2557,8 @@ void Resolver::sanityCheck(core::GlobalState &gs, vector<ast::ParsedFile> &trees
 
 ast::ParsedFilesOrCancelled Resolver::runIncremental(core::GlobalState &gs, vector<ast::ParsedFile> trees) {
     auto workers = WorkerPool::create(0, gs.tracer());
-    trees = ResolveConstantsWalk::resolveConstants(gs, std::move(trees), *workers);
-    computeLinearization(gs);
```

Comment:
I removed linearization from the fast path. It should be a no-op as mixins should not change.

Reply:
can we enforce that?

Reply:
Sure. I can enforce that.

Reply:
Enforce added.